src/main/java/no/nav/foreldrepenger/oversikt/oppslag/MineArbeidsforholdTjeneste.java

```diff
@@ -35,38 +39,58 @@ public MineArbeidsforholdTjeneste() {
 
     @Inject
     public MineArbeidsforholdTjeneste(ArbeidsforholdTjeneste arbeidsforholdTjeneste,
-                                      PersonOppslagSystem personOppslagSystem, VirksomhetTjeneste virksomhetTjeneste) {
+                                      PersonOppslagSystem personOppslagSystem,
+                                      VirksomhetTjeneste virksomhetTjeneste) {
         this.arbeidsforholdTjeneste = arbeidsforholdTjeneste;
         this.personOppslagSystem = personOppslagSystem;
-
         this.virksomhetTjeneste = virksomhetTjeneste;
     }
 
     public List<EksternArbeidsforholdDto> brukersArbeidsforhold(Fødselsnummer brukerFødselsnummer) {
         // Slår opp i Aa-register, velger typer arbeidsforhold som er relevante og mapper om til eksternt format (med navn)
-        return arbeidsforholdTjeneste.finnAktiveArbeidsforholdForIdent(brukerFødselsnummer).stream()
-            .map(this::tilEksternArbeidsforhold)
+        var alleAktiveArbeidsforhold = arbeidsforholdTjeneste.finnAktiveArbeidsforholdForIdent(brukerFødselsnummer);
+        return slåSammenLikeArbeidsforhold(alleAktiveArbeidsforhold).stream()
             .sorted(Comparator.comparing(EksternArbeidsforholdDto::arbeidsgiverNavn))
             .toList();
     }
 
-    public List<EksternArbeidsforholdDto> brukersFrilansoppdragSisteSeksMåneder(Fødselsnummer brukerFødselsnummer) {
-        // Slår opp i Aa-register, velger typer arbeidsforhold som er relevante og mapper om til eksternt format (med navn)
-        return arbeidsforholdTjeneste.finnFrilansForIdent(brukerFødselsnummer).stream()
-            .map(this::tilEksternArbeidsforhold)
-            .sorted(Comparator.comparing(EksternArbeidsforholdDto::arbeidsgiverNavn))
+    private List<EksternArbeidsforholdDto> slåSammenLikeArbeidsforhold(List<Arbeidsforhold> alleArbeidsforhold) {
+        return alleArbeidsforhold.stream()
+            .collect(Collectors.groupingBy(Arbeidsforhold::arbeidsforholdIdentifikator))
+            .values()
+            .stream()
+            .map(this::slåSammeArbeidsforholdFraSammeArbeidsgiver)
+            .flatMap(Collection::stream)
+            .toList();
+    }
+
+    private List<EksternArbeidsforholdDto> slåSammeArbeidsforholdFraSammeArbeidsgiver(List<Arbeidsforhold> arbeidsforholdListe) {
+        var arbeidsforholdIdentifikator = arbeidsforholdListe.getFirst().arbeidsforholdIdentifikator();
+        return arbeidsforholdListe.stream()
```

Comment:
Det vi bør høre med PE om er følgende: Hvis det er disjoint ansettelsesperioder for samme arbeidsgiverid (samme org + arbforholdId) - skal vi da vise ett innslag for hver gruppe ansettelsesperiode, eller slå disse sammen til en sammenhengende sak.
Det kan være man ønsker ulik funksjon for arbeid og frilans
Frilans er enten sammenhengede, eller flere disjointe. Frilans har ikke nødvendigvis stillingsprosent ....